src/flexbox/nasa.css

```diff
@@ -0,0 +1,183 @@
+body {
+  font-family: Helvetica Neue, Arial, sans-serif;
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+  background-color: #E6E6E6;
+}
+
+/*** NAV ***/
+nav {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: auto;
+  padding: 10px 0;
+  overflow: hidden;
+  background-color: #E6E6E6;
+  position: fixed; /* Set the navbar to fixed position */
+  top: 0; /* Position the navbar at the top of the page */
+  width: 100%; /* Full width */
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  justify-content: flex-end;
+  align-items: center;
+  flex-grow: 1;
+}
+
+nav li {
+  padding: 0px 20px;
+}
+
+.search-outer {
+  order: 3;
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 5%;
+}
+
+.search-inner {
+  overflow: hidden;
+  display: inline-flex;
+  padding: 0;
+  margin: 0;
+}
+
+.search-box {
+  border: 0;
+  text-transform: capitalize;
+  padding: 0.3rem 0.3rem 0.3rem 0;
+  font-size: .8em;
+  background: #fff;
+}
+
+.fa-search {
+  padding: 0.3rem;
+}
+
+.search-inner button {
+  background: #205B8A;
+  color: white;
+  border: none;
+  padding: 0.3rem;
+}
+
+/*** MAIN ***/
+
+.nasa-img {
+  background: url(https://www.nasa.gov/sites/default/files/thumbnails/image/potw1636a.jpg) no-repeat center center fixed;
```

Comment:
quotes around link. since this is a link, we're relying on great wifi and/or amazing mobile service. to be safe, it's best to download and resize. no need to change it though!